lib/plugins/Label_H1_StarPOS.ts

```diff
@@ -0,0 +1,49 @@
+import { DateTimeUtils } from '../DateTimeUtils';
+import { DecoderPlugin } from '../DecoderPlugin';
+import { DecodeResult, Message, Options } from '../DecoderPluginInterface';
+import { CoordinateUtils } from '../utils/coordinate_utils';
+import { ResultFormatter } from '../utils/result_formatter';
+
+export class Label_H1_StarPOS extends DecoderPlugin {
+  name = 'label-h1-star-pos';
+  qualifiers() { // eslint-disable-line class-methods-use-this
+    return {
+      labels: ['H1'],
+      preambles: ['*POS'],
+    };
+  }
+
+  decode(message: Message, options: Options = {}) : DecodeResult {
+    let decodeResult = this.defaultResult();
+    decodeResult.decoder.name = this.name;
+    decodeResult.formatted.description = 'Position Report';
+    decodeResult.message = message;
+
+    const msg = message.text;
+    // assuming fixed length until other variants found
+    if (msg.length !== 43 || !msg.startsWith('*POS')) {
+      if (options.debug) {
+        console.log(`Decoder: Unknown H1 message: ${msg}`);
+      }
+      ResultFormatter.unknown(decodeResult, msg);
+      decodeResult.decoded = false;
+      decodeResult.decoder.decodeLevel = 'none';
+      return decodeResult;
+    }
+
+    decodeResult.raw.month = Number(msg.substring(4, 6));
+    decodeResult.raw.day_of_month = Number(msg.substring(6, 8));
```

Comment:
using `day_of_month` to be consistent - thinking of refactoring so they're all just `day` - do we want these as a formatted result too? or is it good as-is?

Reply:
going to leave as-is - we can come back making all messages more consistent